### ISSUE TITLE
Help please ran the code and gave an error    pip install -r requirements.txt 

### ISSUE THREAD

**Issue by Viktorianskiy** [wiseman/py-webrtcvad#81]
ERROR: Command errored out with exit status 1:
   command: 'C:\Users\DappExpert\anaconda3\envs\clone\python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\DAPPEX~1\\AppData\\Local\\Temp\\pip-install-rnoq_ipo\\webrtcvad_66a6a7f27ca44e97b589eb9ceb91fe4a\\setup.py'"'"'; __file__='"'"'C:\\Users\\DAPPEX~1\\AppData\\Local\\Temp\\pip-install-rnoq_ipo\\webrtcvad_66a6a7f27ca44e97b589eb9ceb91fe4a\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d 'C:\Users\DAPPEX~1\AppData\Local\Temp\pip-wheel-9__nm872'
       cwd: C:\Users\DAPPEX~1\AppData\Local\Temp\pip-install-rnoq_ipo\webrtcvad_66a6a7f27ca44e97b589eb9ceb91fe4a\
  Complete output (19 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-3.6
  copying webrtcvad.py -> build\lib.win-amd64-3.6
  running build_ext
  building '_webrtcvad' extension
  creating build\temp.win-amd64-3.6
  creating build\temp.win-amd64-3.6\Release
  creating build\temp.win-amd64-3.6\Release\cbits
  creating build\temp.win-amd64-3.6\Release\cbits\webrtc
  creating build\temp.win-amd64-3.6\Release\cbits\webrtc\common_audio
  creating build\temp.win-amd64-3.6\Release\cbits\webrtc\common_audio\signal_processing
  creating build\temp.win-amd64-3.6\Release\cbits\webrtc\common_audio\vad
  C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -D_WIN32 -Icbits -IC:\Users\DappExpert\anaconda3\envs\clone\include -IC:\Users\DappExpert\anaconda3\envs\clone\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\include" /Tccbits\pywebrtcvad.c /Fobuild\temp.win-amd64-3.6\Release\cbits\pywebrtcvad.obj
  pywebrtcvad.c
  C:\Users\DappExpert\anaconda3\envs\clone\include\pyconfig.h(59): fatal error C1083: ЌҐ г¤ Ґвбп ®вЄалвм д ©« ўЄ«озҐ­ЁҐ: io.h: No such file or directory,
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
  ----------------------------------------
  ERROR: Failed building wheel for webrtcvad
  Running setup.py clean for webrtcvad
Failed to build webrtcvad
Installing collected packages: zipp, typing-extensions, urllib3, threadpoolctl, scipy, pyparsing, llvmlite, joblib, importlib-metadata, idna, dataclasses, charset-normalizer, cached-property, werkzeug, scikit-learn, requests, protobuf, packaging, numba, markdown, jsonpointer, importlib-resources, h5py, grpcio, colorama, appdirs, absl-py, wrapt, websocket-client, tqdm, tornado, torchfile, termcolor, tensorflow-estimator, tensorboard, soundfile, resampy, pyzmq, python-dateutil, PyQt5-sip, PyQt5-Qt5, pynndescent, pooch, pillow, kiwisolver, keras-preprocessing, keras-applications, jsonpatch, google-pasta, gast, dill, decorator, cycler, audioread, astor, webrtcvad, visdom, Unidecode, umap-learn, tensorflow-gpu, sounddevice, PyQt5, multiprocess, matplotlib, librosa, inflect, distance
    Running setup.py install for webrtcvad ... error
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\DappExpert\anaconda3\envs\clone\python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\DAPPEX~1\\AppData\\Local\\Temp\\pip-install-rnoq_ipo\\webrtcvad_66a6a7f27ca44e97b589eb9ceb91fe4a\\setup.py'"'"'; __file__='"'"'C:\\Users\\DAPPEX~1\\AppData\\Local\\Temp\\pip-install-rnoq_ipo\\webrtcvad_66a6a7f27ca44e97b589eb9ceb91fe4a\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\DAPPEX~1\AppData\Local\Temp\pip-record-p11aszvu\install-record.txt' --single-version-externally-managed --compile --install-headers 'C:\Users\DappExpert\anaconda3\envs\clone\Include\webrtcvad'
         cwd: C:\Users\DAPPEX~1\AppData\Local\Temp\pip-install-rnoq_ipo\webrtcvad_66a6a7f27ca44e97b589eb9ceb91fe4a\
    Complete output (19 lines):
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.6
    copying webrtcvad.py -> build\lib.win-amd64-3.6
    running build_ext
    building '_webrtcvad' extension
    creating build\temp.win-amd64-3.6
    creating build\temp.win-amd64-3.6\Release
    creating build\temp.win-amd64-3.6\Release\cbits
    creating build\temp.win-amd64-3.6\Release\cbits\webrtc
    creating build\temp.win-amd64-3.6\Release\cbits\webrtc\common_audio
    creating build\temp.win-amd64-3.6\Release\cbits\webrtc\common_audio\signal_processing
    creating build\temp.win-amd64-3.6\Release\cbits\webrtc\common_audio\vad
    C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -D_WIN32 -Icbits -IC:\Users\DappExpert\anaconda3\envs\clone\include -IC:\Users\DappExpert\anaconda3\envs\clone\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\include" /Tccbits\pywebrtcvad.c /Fobuild\temp.win-amd64-3.6\Release\cbits\pywebrtcvad.obj
    pywebrtcvad.c
    C:\Users\DappExpert\anaconda3\envs\clone\include\pyconfig.h(59): fatal error C1083: ЌҐ г¤ Ґвбп ®вЄалвм д ©« ўЄ«озҐ­ЁҐ: io.h: No such file or directory,
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'C:\Users\DappExpert\anaconda3\envs\clone\python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\DAPPEX~1\\AppData\\Local\\Temp\\pip-install-rnoq_ipo\\webrtcvad_66a6a7f27ca44e97b589eb9ceb91fe4a\\setup.py'"'"'; __file__='"'"'C:\\Users\\DAPPEX~1\\AppData\\Local\\Temp\\pip-install-rnoq_ipo\\webrtcvad_66a6a7f27ca44e97b589eb9ceb91fe4a\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\DAPPEX~1\AppData\Local\Temp\pip-record-p11aszvu\install-record.txt' --single-version-externally-managed --compile --install-headers 'C:\Users\DappExpert\anaconda3\envs\clone\Include\webrtcvad' Check the logs for full command output.
